learners/BF_measles.Rmd

```diff
@@ -0,0 +1,894 @@
+    approximately $N\approx 22.67$ million.
+
+2.  **Population Structure**:
+
+    -   **No Age Stratification**: Assume the population is homogeneous
+        with respect to age. This means that all individuals, regardless
+        of age, have the same susceptibility to measles and contribute
+        equally to the transmission dynamics.
+
+3.  **Initial Conditions**:
+
+    -   A single infectious case is introduced into the population:
+        $I_0 = 1$.
+    -   The entire population, except for this one case, is initially 
+    susceptible: $S_0 = N - 1$. This 
+    assumption simplifies the model and allows us to explore the spread of 
+    infection in the absence of immunity. Although real populations typically 
+    have some immunity due to vaccination or prior infection.
+    -   No individuals are in the exposed stage at the start: $E_0 = 0$.
+    -   No individuals have recovered: $R_0 = 0$.
+
+4.  **latent Period**: The average latent period for measles is
+    around 8 days (the time period between exposure and becoming
+    infectious), so the transition rate from exposed to infectious
+    individuals is: $\omega = \frac{1}{8}$.
+
+5.  **Infectious Period**: The average infectious period lasts for 5
+    days, which means the recovery rate is: $\gamma = \frac{1}{5}$.
+
+6.  **Basic Reproduction Number** $\mathcal{R}_0$:
+
+    -   $\mathcal{R}_0 =15$. For measles this value typically ranges
+        from 12 to 18, or even more [Fiona, et al](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(17)30307-9/abstract).
+    -   This makes the transmission rate
+        $\beta = \mathcal{R}_0 \gamma = 3$--meaning each infectious individual causes 3 new infections
+        per day in a fully susceptible population.
+
+By applying these assumptions and parameters to
+[Equation 1](#eq-s1), we can simulate the potential
+trajectory of the measles outbreak using the syntax provided by the
+{epidemics} package as shown in the below is code chunk.
+
+```{r,warning=FALSE,message=FALSE}
+# Loading libraries
+library(dplyr)
+library(ggplot2)
+library(epidemics)
+```
+
+
+```{r}
+# Define the population structure for Burkina Faso
+bfaso <- epidemics::population(
+  name = "Burkina Faso",
+  contact_matrix = matrix(1), # Well-mixed population
+  demography_vector = 22.67e6, # Population size
+  # Initial conditions: Susceptible, exposed, Infectious, vaccinated, recovered
+  initial_conditions = matrix(
+    c(1 - 1 / 22.67e6, 1 / 22.67e6, 0, 0, 0),
+    nrow = 1, ncol = 5
+  )
+)
+
+# Run the SEIR model simulation
+output <- epidemics::model_default(
+  population = bfaso,
+  transmission_rate = 15 / 5, # R0/infectious period = 15/5 = 3
+  infectiousness_rate = 1 / 8, # 1/latent period (pre-infectious phase)
+  recovery_rate = 1 / 5, # 1/infectious period
+  time_end = 120, # Simulation over 120 days
+  increment = 1 # Daily increments
+)
+
+# Plot the epidemic curve for susceptible, exposed, infectious, and recovered
```

Comment:
@Degoot-AM this episode fits nicely as the summative for the session on intro to idm. For the training session, we can do some sections by livecoding and others by copy/paste. I created this summative code with labels for each https://github.com/avallecam/epicatador/blob/main/74-epidemics-bf-measles-summative.R

In that summative, I propose to make simpler plots (e.g., whole aggregates of new infections). I prefer to keep ggplot2 code as simple as possible (geom only as much), then less noise when showcasing {epidemics}. But better to keep that edit to other PR.